carma/launch/carma_src.launch

```diff
@@ -85,26 +71,12 @@
     </include>
   </group>
 
-
```

Comment:
Its not letting me add a comment on the correct line, but the "launch_drivers" and "vehicle_calibration_dir" args don't need to be passed to hardware_interface.launch. Please also remove reference to those arguments in  hardware_interface.launch.

Reply:
removed all references of drivers.launch and launch_drivers